android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -2294,12 +2297,31 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         pageLoaded(view, url);
       } else if (url.indexOf("hideKeyboard") >= 0) {
         if (KMTextView.activeView != null && KMTextView.activeView.getClass() == KMTextView.class) {
+          InAppKeyboard.dismissHelpBubble();
           KMTextView textView = (KMTextView) KMTextView.activeView;
           textView.dismissKeyboard();
         }
       } else if (urlCommand.getQueryParameter("globeKeyAction") != null) {
+        InAppKeyboard.dismissHelpBubble();
+        InAppKeyboard.setShouldShowHelpBubble(false);
+
+        // // Original had logic that would shortcut this in some scenarios, but I'm unclear as to its real purpose.
+        // if (!InAppKeyboard.isHelpBubbleEnabled) {
+        //   return false;
```

Comment:
Good question. It's been there since the original version 2.8
https://github.com/keymanapp/keyman/blob/2b3fa63cea49f47201226d74d35ad57f25d2385e/android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java#L1900-L1902

I'm not sure if the handling of help bubble and globe key action somehow got conflated